diff --git a/src/collection/backend/lmdb.cc b/src/collection/backend/lmdb.cc
index 1ffca80..62a502d 100644
--- a/src/collection/backend/lmdb.cc
+++ b/src/collection/backend/lmdb.cc
@@ -510,7 +510,7 @@ MDBEnvProvider::MDBEnvProvider() : m_env(NULL), valid(false) {
     int rc;
     MDB_txn *txn;
     mdb_env_create(&m_env);
-    rc = mdb_env_open(m_env, "./modsec-shared-collections",
+    rc = mdb_env_open(m_env, "/run/modsec-shared-collections",
         MDB_WRITEMAP | MDB_NOSUBDIR, 0664);
 
     if (rc == 0) {
